src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
+
+  switch (true) {
+    case days < 3:
+      return sum;
+
+    case days >= 3 && days < 7:
+      sum -= 20;
+
+      return sum;
+
+    case days >= 7:
+      sum -= 50;
+
+      return sum;
+  }
 }
```

Comment:
let's create variables for all these numbers, to understand what is going on here
<img width="1029" alt="image" src="https://user-images.githubusercontent.com/102413088/182326610-925d0649-6bb3-4ead-835c-02adf2e0c11f.png">